docs/source/modeling-transactions.rst

```diff
@@ -33,11 +43,16 @@ To do so, we create ``T1`` as follows.
 			]
 	}
 
-``T1`` redeems ``T``, providing 42 as witness. The witness is evaluated against the script in the output of ``T``,
-i.e. :code:`fun(x) . x==42`. But, differently from the previous example, the transaction splits the bitcoins
+To redeem  the money of ``T`` we must provide the number 42 as *witness* (in the input field next to the transaction we want to redeem), so that the output script of ``T``  i.e. :code:`fun(x) . x==42` would evaluate to true. 
+
+But, differently from the previous example, the transaction splits the bitcoins
 it redeems between two outputs. The first contains 20 BTC and can be redeemed whit a witness different from 0,
 while the other contains 30 BTC and can be redeemed solving a simple equation.
 
+
+"""""""""""""""""""""""""""""""
+Double input  
+"""""""""""""""""""""""""""""""
 This time we want to redeem our bitcoins and secure them in a transaction that can be spent only by us. 
 
 .. code-block:: btm
```

Comment:
Suggestion: use an address instead the private key ``k`` and fix the prose accordingly.
The example could be:
```
const myAddress = wif:n3s2D2TSroNtMdAFV88YfhExUmNidqesD1

transaction T2 {
        input = [
                T1@0: 1;
                T1@1: 2 0
                ]
        output = 50 BTC: fun(x) . versig(myAddress; x)
}
```